website/docs/whats-new/release-notes/partials/figma-library-components.md

```diff
@@ -65,10 +90,6 @@ Added new components:
 - `Accordion`
 - `MaskedInput`
 
-### July 12th, 2023
-
-Added a “Form Primitives” page to house the `Fieldset` component.
-
```

Comment:
Why is this entry showing as deleted?

Reply:
Good question! The script that converts the changelog entries to website content was [written to keep only a subset of entries](https://github.com/hashicorp/design-system/blob/main/website/lib/changelog/generate-changelog-markdown-files.mjs#L83-L100) presumably to keep only the latest, relevant entries.